pkg/nimble/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME    = nimble
 PKG_URL     = https://github.com/apache/mynewt-nimble.git
```

Comment:
If we are temporarily switching the upstream needs to be changed

Reply:
I don't think so, as the fork is PRed, it seems like git is fine in finding the correct branch by the provided hash

Reply:
Not for me:

```
make -C examples/nimble_scanner/ distclean clean flash term
rm -rf /home/francisco/workspace/RIOT/examples/nimble_scanner/bin/nrf52dk/pkg-build/nimble
rm -rf /home/francisco/workspace/RIOT/build/pkg/nimble
rm -rf /home/francisco/workspace/RIOT/examples/nimble_scanner/bin/nrf52dk/pkg-build/nimble
Building application "nimble_scanner" for "nrf52dk" with MCU "nrf52".

[INFO] cloning nimble
Cloning into '/home/francisco/workspace/RIOT/build/pkg/nimble'...
remote: Enumerating objects: 140, done.
remote: Counting objects: 100% (140/140), done.
remote: Compressing objects: 100% (105/105), done.
remote: Total 41218 (delta 54), reused 66 (delta 29), pack-reused 41078
Receiving objects: 100% (41218/41218), 12.28 MiB | 10.42 MiB/s, done.
Resolving deltas: 100% (25153/25153), done.
fatal: reference is not a tree: 7a2bf2420a1a2bcd8bbb5bf8f5927882b3c499c0
make[1]: *** [/home/francisco/workspace/RIOT/pkg/pkg.mk:123: /home/francisco/workspace/RIOT/build/pkg/nimble/.git] Error 128
make: [/home/francisco/workspace/RIOT/examples/nimble_scanner/../../Makefile.include:696: pkg-prepare] Error 2 (ignored)
```

Reply:
This still needs addressing.

Reply:
Once this is addressed please trigger the ci.